pkgs/development/libraries/libdeltachat/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, openssl
+, perl
+, pkg-config
+, rustPlatform
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libdeltachat";
+  version = "1.54.0";
+
+  src = fetchFromGitHub {
+    owner = "deltachat";
+    repo = "deltachat-core-rust";
+    rev = version;
+    sha256 = "02hvsfv1yar8bdpkfrfiiicq9qqnfhp46v6qqph9ar6khz3f1kim";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "1p5yrhczp9nfijbvkmkmx1rabk5k3c1ni4k1vc0mw4jgl26lslcm";
+  };
```

Comment:
just curious as to why `rustPlatform.buildRustPackage` wasn't used since, it looks like you're running most of the logic.

And maybe just comment as to why `buildRustPackage` isn't suitable if there is a good reason :)

Reply:
Cause I'm using CMake. Using `buildRustPackage` leads to an empty output though I guess I could do `make` and `make install` in `postBuild` and `installPhase`. I don't see any advantage of that though.

Reply:
@jonringer Do you know of an easier way?

Reply:
Ah, I missed that. No, the way that you did it makes sense then :)